zeroconf/test.py

```diff
@@ -716,6 +756,29 @@ def test_invalid_addresses(self):
                 addresses=[addr],
             )
 
+    def test_service_info_for_local_name(self):
+        type_ = "_hap._tcp.local."
+        registration_names = (
+            "Rachio-C73233.local.",
+            'YeelightColorBulb-3AFD.local.',
+            'YeelightTunableBulb-7220.local.',
+            "AlexanderHomeAssistant 74651D.local.",
+            'iSmartGate-152.local.',
+            'MyQ-FGA.local.',
+            'lutron-02c4392a.local.',
+            'WICED-hap-3E2734.local.',
+            'MyHost.local.',
+        )
+
+        for registration_name in registration_names:
+            ServiceInfo(
+                type_,
+                registration_name,
+                port=80,
+                addresses=[socket.inet_aton("10.0.1.2")],
+                strict=False,
+            )
```

Comment:
I don't think we want to test constructing `ServiceInfo` like this (see my comment regarding `strict` in `ServiceInfo` above) and it's rather redundant anyway. This test can just go IMO.

Reply:
removed